core/source/Http.mint

```diff
@@ -152,7 +152,9 @@ module Http {
     |> Http.send()
   */
   fun jsonBody (body : Object, request : Http.Request) : Http.Request {
-    { request | body = `JSON.stringify(#{body})` }
+    { request |
+      body = `JSON.stringify(#{body})`,
+      headers = Array.push(`new Record({ value: "application/json", key: "content-type" })`, request.headers) }
```

Comment:
```suggestion
      headers = Array.push(`new Record({ value: "application/json", key: "Content-Type" })`, request.headers) }
```

Reply:
This crossed my mind too, but it appears to be [case-insensitive](https://stackoverflow.com/a/7718542/2513269). Or is this to improve the readability of the code?

Reply:
I suggested it due to the conventional form in writing.

Reply:
Point taken 👍